doc/content/getting-started/migrating/migrate-from-chirpstack.md

```diff
@@ -27,9 +27,16 @@ $ export FREQUENCY_PLAN_ID="EU_863_870"         # Set The Things Stack Frequency
 $ export CHIRPSTACK_API_INSECURE=0              # Set to 1 if not using TLS on ChirpStack
 ```
 
-{{< note >}} `JoinEUI` and `FrequencyPlanID` have to be set because ChirpStack does not store these variables. 
+{{< note >}} If using Windows OS, replace `export` with `set` and remove the double-quotes in commands above. For example, you would use:
+
+```bash
+$ set CHIRPSTACK_API_TOKEN=7F0as987e61...
+```
+{{</ note >}}
+
+{{< info >}} `JoinEUI` and `FrequencyPlanID` have to be set because ChirpStack does not store these variables. 
```

Comment:
Just use note. There's no value added from the info shortcode. It mostly exists to be used by other shortcodes, like the CLI only shortcode.

```suggestion
{{< note >}} `JoinEUI` and `FrequencyPlanID` have to be set because ChirpStack does not store these variables. 
```